ts/pages/account/account_voting_power_overview.tsx

```diff
@@ -0,0 +1,182 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+import { Button } from 'ts/components/button';
+import { CircleCheckMark } from 'ts/components/ui/circle_check_mark';
+import { generateUniqueId, Jazzicon } from 'ts/components/ui/jazzicon';
+import { PanelHeader } from 'ts/components/ui/panel_header';
+import { StatFigure } from 'ts/components/ui/stat_figure';
+import { State } from 'ts/redux/reducer';
+import { AccountReady } from 'ts/types';
+
+import { colors } from 'ts/style/colors';
+import { formatZrx } from 'ts/utils/format_number';
+
+interface VotingPowerOverviewProps {
+    name: string;
+    operatorAddress: string;
+    poolId: string;
+    delegation: number;
+    isVerified: boolean;
+    websiteUrl?: string;
+    logoUrl?: string;
+}
+
+interface SelfVotingPowerOverviewProps {
+    delegation: number;
+    isSelfDelegated: boolean;
+    onMoveStake: () => void;
+}
+
+export const AccountVotingPowerOverview: React.StatelessComponent<VotingPowerOverviewProps> = ({
+    name,
+    websiteUrl,
+    logoUrl,
+    operatorAddress,
+    poolId,
+    isVerified,
+    delegation,
+}) => {
+    return (
+        <Wrap>
+            <Flex>
+                <PanelHeader
+                    subtitle={websiteUrl}
+                    avatarSrc={logoUrl}
+                    isResponsiveAvatar={true}
+                    address={operatorAddress}
+                    poolId={poolId}
+                >
+                    <Title>
+                        {name}
+                        {isVerified && <CircleCheckMark />}
+                    </Title>
+                </PanelHeader>
+
+                <Stats>
+                    <StatFigure label="Your Delegation" value={formatZrx(delegation).formatted} isNoBorder={true} shouldShowZrxLabel={true} />
+                </Stats>
+            </Flex>
+        </Wrap>
+    );
+};
+
+export const AccountSelfVotingPowerOverview: React.StatelessComponent<SelfVotingPowerOverviewProps> = ({
```

Comment:
i think StatelessComponent type might be deprecated

Reply:
Changed it to `React.FC`